common/logging/logger.py

```diff
@@ -0,0 +1,64 @@
+import logging
```

Comment:
sync with Asad about changes in the logger. Either remove it from here [his branch will be merged later] or include his changes here.